platform/darwin/src/MGLLight.h

```diff
@@ -58,6 +58,9 @@ NS_INLINE MGLSphericalPosition MGLSphericalPositionMake(CGFloat radial, CLLocati
 
 /**
  An `MGLLight` object represents the light source for extruded geometries in `MGLStyle`.
+
+#### Related examples
+See the <a href="https://www.mapbox.com/ios-sdk/maps/examples/light-example/">Adjust light of 3D buildings</a> to learn how to create and modify the light source for 3D geometries.
```

Comment:
Hrm, this particular entry in the generated header/docs doesn’t appear to auto-wrap... 🤔

Reply:
Yeah, not sure why that is... Seems like digging through [generate-style-code.js](https://github.com/mapbox/mapbox-gl-native/blob/master/platform/darwin/scripts/generate-style-code.js) at this point is a bit tangential for this PR. I'm making a note to dig through it separately and I'll open up a new issue later this week.

Reply:
Call `wrap()` on `doc`:

https://github.com/mapbox/mapbox-gl-native/blob/35a0811eeb7b27378f1e37d942bfa4df2f44c6a8/platform/darwin/src/MGLLight.h.ejs#L66

like this:

https://github.com/mapbox/mapbox-gl-native/blob/35a0811eeb7b27378f1e37d942bfa4df2f44c6a8/platform/darwin/src/MGLLight.h.ejs#L74